pyproject.toml

```diff
@@ -33,6 +33,10 @@ types-toml = "^0.10.8"
 questionary = "^1.10.0"
 certifi = "^2022.12.7"
 psutil = "^5.9.4"
+types-urllib3 = "^1.26.25.10"
```

Comment:
Bit of a nitpick, but we only need types in dev

Reply:
Hmmm, that what i thought. I initially placed them in tool.poetry.dev-dependencies https://github.com/statisticsnorway/ssb-project-cli/pull/237/commits/129f87175192f6247b927383c66512a5eeb99973

But the tests failed https://github.com/statisticsnorway/ssb-project-cli/actions/runs/4785414366/jobs/8508096835

Also tried to move them to tool.poetry.group.dev.dependencies https://github.com/statisticsnorway/ssb-project-cli/pull/237/commits/a750d07a4dcbc9ac7efd63dd92208a4f3d15e526 

The tests still failed https://github.com/statisticsnorway/ssb-project-cli/actions/runs/4785509401/jobs/8508313059